community/modules/scripts/spack-install/scripts/install_spack_deps.yml

```diff
@@ -18,7 +18,7 @@
   become: yes
   hosts: localhost
   tasks:
-  - name: Install pip3 and git
+  - name: Install git
```

Comment:
It looks like pip is still being installed. Maybe update this to something more generic like "Install dependencies through the package manager"?

Reply:
Done